CMakeLists.txt

```diff
@@ -233,6 +263,12 @@ include(music_gme)
 include(music_cmd)
 
 option(USE_MIDI            "Build with MIDI support" ON)
+
+if(VITA)
+    set(USE_MIDI OFF)
+    set(USE_GME OFF)
```

Comment:
Keep the note that GME is required for default SMBX assets for TheXTech as there are lot of SPC files being played, and one NSF file too. MIDI is also required for Adventure of Demo and for my mod for "Lowswer's Conquest DEMO 2013" where I replaced the lot of music with various MIDI using libADLMIDI and libOPNMIDI synthesizers
(to reduce the CPU load, do use the MAME YM2612 or even GENS (but I suggest to use NP2 as better stability) on libOPNMIDI and DosBox OPL3 emulator on libADLMIDI as they were fastest)